.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: go
 go:
-  - 1.5
+  - 1.7
```

Comment:
You mean be explicit about the patch version? ("1.7" uses the latest 1.7 release)